core/src/main/R/base/strwrap.R

```diff
@@ -196,7 +196,9 @@ trimws <- function(x, which = c("both", "left", "right"),
                    whitespace = "[ \t\r\n]")
 {
     which <- match.arg(which)
-    mysub <- function(re, x) sub(re, "", x, perl = TRUE)
+    mysub <- function(re, x) {
```

Comment:
By "inline function", I meant a function defined in another function. But that's ok, the optimizer needs to handle inline functions at some point anyway.

Reply:
I think I misread it as "one line function."